README.md

```diff
@@ -55,8 +55,72 @@ Our Developer Success Team can assist with answering any questions as well as pr
 
 [Forums, Support and Knowledge Base](https://community.playfab.com/hc/en-us)
 
+6. Example Usage (Template):
+----
+
+```
+// In this javascript example, it is assumed your html will include the PlayFab JavaScript Client SDK
+// For example, in your html header, add this:
+// <script src="https://download.playfab.com/PlayFabClientApi.js"></script>
+// (This cannot easily be done from a .js file)
+
+// You must always set your titleId first
+PlayFab.settings.titleId = null; // Set your titleId here.  Example: "6195"
```

Comment:
Might double check to make sure the PlayFab file loaded. Its rare, but the CDN can fail.

//something like:
if(!PlayFab)
            {
              OutputError("The PlayFab SDK could not be found. Double check your script sources");
              return;
            }